kedro-datasets/kedro_datasets/netcdf/netcdf_dataset.py

```diff
@@ -0,0 +1,218 @@
+"""NetCDFDataSet loads and saves data to a local netcdf (.nc) file."""
+import logging
+import os
+from copy import deepcopy
+from glob import glob
+from pathlib import Path, PurePosixPath
+from typing import Any, Dict
+
+import fsspec
+import xarray as xr
+from kedro.io.core import (
+    AbstractDataset,
+    DataSetError,
```

Comment:
Please use the renamed `DatasetError` instead. 
```suggestion
    DatasetError,
```

Reply:
Whoops, I misread the suggestions you had gave earlier. I thought the deprecation was the other way around. Will update that now.